frontend/packages/topology/src/components/quick-search/QuickSearchContent.scss

```diff
@@ -0,0 +1,13 @@
+.odc-quick-search-content {
+  min-height: 390px;
+  max-height: 390px;
```

Comment:
🤔  is this not just `height: 390px` -- am I missing other styles that make use of `min/max` heights?

Reply:
So when I used just the height somehow the container was shrinking when there were less than 6 items, not sure if it was a browser issue but now it seems to work fine.